hack/test-cmd.sh

```diff
@@ -133,16 +133,16 @@ openshift start \
   --images="${USE_IMAGES}"
 
 # validate config that was generated
-[ "$(openshift ex validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
-[ "$(openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml 2>&1 | grep SUCCESS)" ]
+[ "$(oadm validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
```

Comment:
I'm not sure it should be a top-level command. Maybe we should also promote `diagnostics` and make it under `oadm diagnostics validate`. Also what's the difference of these `validate` commands and `openshift ex diagnostics --master-config=''`?